zipline/gens/brokers/ib_broker.py

```diff
@@ -456,9 +459,11 @@ def fundamentalData(self, req_id, data):
     def marketDataType(self, req_id, market_data_type):
         log_message('marketDataType', vars())
 
-    def realtimeBar(self, req_id, time, open_, high, low, close, volume, wap,
+    def realtimeBar(self, req_id, time, open, high, low, close, volume, wap,
```

Comment:
open() is a python function. naming the parameter the same shadows its declaration. Could you please change the variable back to open_ ?